Source/Renderer/RenderState.js

```diff
@@ -173,12 +182,12 @@ define([
             (!defined(viewport.width)) ? context.drawingBufferWidth : viewport.width,
             (!defined(viewport.height)) ? context.drawingBufferHeight : viewport.height) : undefined;
 
+        //>>includeStart('debug', pragmas.debug);
```

Comment:
Actually, I would say this `RuntimeError` and the one below could really be `DeveloperError`s.